mailpoet/assets/js/src/wizard/steps/woocommerce_step.tsx

```diff
@@ -30,7 +30,7 @@ function WizardWooCommerceStep({
     if (importType === null || allowed === null) {
       return false;
     }
-    submitForm(importType, allowed === 'true');
+    submitForm(importType, allowed === true);
```

Comment:
I've encountered this normalization issue that's why [this utility function was introduced](https://github.com/mailpoet/mailpoet/blob/66dc6e67ab7506a69e1d7d1a29ff43a7cff431eb/mailpoet/assets/js/src/common/utils.ts#L2), maybe we can use it here as well?

Reply:
I wanted to use the function, but I see that the component returns a boolean, so I can use the value directly.